disnake/http.py

```diff
@@ -1833,6 +1833,86 @@ def delete_stage_instance(
             Route("DELETE", "/stage-instances/{channel_id}", channel_id=channel_id), reason=reason
         )
 
+    # Scheduled event management
+    # TODO: type hints for dicts
+
+    def get_guild_scheduled_events(self, guild_id: int, with_user_count: bool = None):
+        params = {}
+
+        if with_user_count is not None:
+            params["with_user_count"] = with_user_count
+
+        return self.request(
+            Route("GET", "/guilds/{guild_id}/events", guild_id=guild_id), params=params
+        )
+
+    def create_guild_scheduled_event(
+        self,
+        guild_id: int,
+        *,
+        name: str,
+        privacy_level: Dict[str, Any],
```

Comment:
If I'm reading the source correctly this should be an `int` (same thing on `edit_guild_scheduled_event`)